doc/files.md

```diff
@@ -56,7 +56,8 @@ Subdirectory       | File(s)               | Description
 `indexes/coinstats/db/` | LevelDB database | Coinstats index; *optional*, used if `-coinstatsindex=1`
 `wallets/`         |                       | [Contains wallets](#multi-wallet-environment); can be specified by `-walletdir` option; if `wallets/` subdirectory does not exist, wallets reside in the [data directory](#data-directory-location)
 `./`               | `anchors.dat`         | Anchor IP address database, created on shutdown and deleted at startup. Anchors are last known outgoing block-relay-only peers that are tried to re-connect to on startup
-`./`               | `banlist.dat`         | Stores the IPs/subnets of banned nodes
+`./`               | `banlist.dat`         | Stores the addresses/subnets of banned nodes (deprecated). `bitcoind` or `bitcoin-qt` no longer save the banlist to this file, but read it on startup if `banlist.json` is not present.
+`./`               | `banlist.json`        | Stores the addresses/subnets of banned nodes.
```

Comment:
@vasild 
In your opinion, when will be an appropriate moment to move the `banlist.dat` description into the "Legacy subdirectories and files" subsection?

Reply:
See https://github.com/bitcoin/bitcoin/pull/20966/commits/cd9da1b2a2f2dcad008e5a1a079a093e77565369#r634452821.

Reply:
Good question! I am not sure. Either now or in a next release where we completely ignore `banlist.dat`?